app/main/helpers/s3.py

```diff
@@ -1,8 +1,11 @@
 import os
 import boto
+import boto.exception
 import datetime
 import mimetypes
 
+from boto.exception import S3ResponseError  # noqa
```

Comment:
That all makes sense - yes definitely keep the `#noqa` there or someone will remove it at some point and break things (in all likelihood me).